client/src/telemetry/glean-context.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import { path, referrer } from "./generated/pageView";
+import { clicked } from "./generated/element";
+import * as pings from "./generated/pings";
+import * as Glean from "@mozilla/glean/web";
+import { GLEAN_CHANNEL, GLEAN_DEBUG, GLEAN_ENABLED } from "../env";
+
+export type PageProps = {
+  referrer: string | undefined;
+  path: string | undefined;
+};
+
+export type PageEventProps = {
+  referrer: string | undefined;
+  path: string | undefined;
+};
+
+export type ElementClickedProps = {
+  source: string;
+  subscription_type: string;
+};
+
+export type GleanAnalytics = {
```

Comment:
Can we call it `GleanTelemetry` (GT) to avoid confusion with Google Analtyics (GA)?

```suggestion
export type GleanTelemetry = {
```

Reply:
I think we colloquially just call it 'Glean' so all good to leave naming the same.